.github/workflows/documentation.yml

```diff
@@ -0,0 +1,48 @@
+name: Documentation
```

Comment:
Can one of you tell me how to test that before merging? The action does not appear in Actions

Reply:
Also here @Scienfitz can probably help. Normally, you would be able to select the new action from the "actions" page and run it against your branch (at least, that's how you can do it on azure). But here, your workflow doesn't even show up on the actions page. Not sure.... but Martin will know

Reply:
you cannot test it before its in main, hence the suggestion to use BayBE-test for pipeline tests

Reply:
welcome to `the pain of pipelines`

maybe try adding the pull-request trigger to the top of the file, theres a chance that you can then run this action. But after everything is tested it should be removed again